apis/python/src/tiledbsoma/io/conversions.py

```diff
@@ -78,10 +80,11 @@ def to_tiledb_supported_array_type(name: str, x: _MT) -> _MT:
     return x
 
 
-def csr_from_coo_table(tbl: pa.Table, num_rows: int, num_cols: int) -> sp.csr_matrix:
+def csr_from_coo_table(
+    tbl: pa.Table, num_rows: int, num_cols: int, context: SOMATileDBContext
+) -> sp.csr_matrix:
     """Given an Arrow Table containing COO data, return a ``scipy.sparse.csr_matrx``."""
```

Comment:
`scipy.sparse.csr_matrix` (I know this typo isn't new on this PR but may as well fix it here/now)